src/Bab/RabbitMq/Response.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Bab\RabbitMq;
+
+class Response
```

Comment:
I don't really understand why you don't directly use the guzzle Response object?